osx/setup.sh

```diff
@@ -303,56 +308,18 @@ if ! [ -d "$KA_LITE_MONITOR_APP_PATH" ]; then
     exit 2
 fi
 
-# sign the .app file
-# unlock the keychain first so we can access the private key
-# security unlock-keychain -p $KEYCHAIN_PASSWORD
-# codesign -s "$SIGNER_IDENTITY_APPLICATION" --force "$KA_LITE_MONITOR_APP_PATH"
```

Comment:
Why did you remove the commented `codesign` command?  Do we not need it anymore?  We use it as a placeholder for now until we need it.